chain/default_store.go

```diff
@@ -118,32 +118,30 @@ func (store *DefaultStore) Load(ctx context.Context) error {
 	logStatusEvery := startHeight / 10
 
 	var genesii types.TipSet
-	err = store.walkChain(ctx, headTs.ToSlice(), func(tips []*types.Block) (cont bool, err error) {
-		if logStatusEvery != 0 && (tips[0].Height%logStatusEvery) == 0 {
-			logStore.Infof("load tipset: %s, height: %v", tips[0].Cid().String(), tips[0].Height)
+	for iterator := IterAncestors(ctx, store, headTs); !iterator.Complete(); err = iterator.Next() {
+		if err != nil {
+			return err
 		}
-		ts, err := types.NewTipSet(tips...)
+		genesii = iterator.Value()
```

Comment:
i find it confusing to call these intermediate tipsets genesii. i prefer the pre-change way of calling them simply ts, which is more accurate. anything but genesii, except for the actual genesis ts.